docs/modular/ansible/index.md

```diff
@@ -218,14 +218,54 @@ The main grid view uses the bottom row for navigation, which are dimly lit into
 
 ![](images/grid_KR_NAVIGATION_1.2.png)
 
-Each *track* has parameters *Trigger, Note, Octave,* and *Duration*. These parameters comprise of the sequence loop.
+### Parameters
+
+Each *track* has four parameters *Trigger, Note, Octave,* and *Duration*. These parameters work together to form the sequence loop.
 
 A *note* will occur only when a *trigger* is set. This note will be the pitch specified by *note* in *octave* of a specified *duration*.
 
 The *trigger* view shows all four tracks at once, whereas all other parameter views display the track selected.
 
 The *duration* view has downward sliders (lower is longer) and a master duration multiplier at the top.
 
+#### Parameter extensions
+
+Many of the parameter views have sub pages that extend or augment their functionality. To enter into each sub page, switch to the parameter using buttons 5-8, then tap the parameter button again. When you are in the parameter extension page, the parameter button will blink, to assist with navigation.
+
+All of the subpages have their own probability, clock division and loop controls, allowing for some very complex phrasing to be built.
+
+* *Trigger* view becomes *Trigger Ratcheting*
+* *Note* view becomes *Alternate Note*
+* *Octave* view becomes *Glide*
+* *Duration* does not currently have a sub page
+
+
+##### Trigger Ratcheting
+The *trigger* view sub page is where you can control the amount of ratcheting per trigger.
+
+To enter the ratcheting page, press the trigger parameter button a second time when you are in the trigger view.
+
+Ratcheting uses rows 3-6 to determine the amount of sub triggers that will fire for each trigger column. Selecting row 6 will cause no extra triggers to fire for that step. Selecting row 5 causes 1 extra trigger to fire, effectively turning an 8th note into two 16th notes. Row 4 will cause 2 extra triggers, for a total of 3, turning the single trigger into a triplet. Finally row 3 will cause 3 extra triggers to fire, turning an 8th note into 4 32nd notes.
+
+![](images\grid_KR_triggerRatcheting.png)
```

Comment:
Is the ratching setting per track? I assume it is, but the "any overlapping note" in the png is confusing.

Reply:
Good call. I'll make sure both the pngs call out the track